x/cdp/module.go

```diff
@@ -26,6 +27,9 @@ var (
 	// _ module.AppModuleSimulation = AppModule{}
 )
 
+// ConsensusVersion defines the current module consensus version.
+const ConsensusVersion = 2
```

Comment:
@nddeluca @pirtleshell should this be 1 since it wasn't defined before or does the first version of the cdp module store point to the current version (pre kava-16)

Reply:
updated